play-json-jsoniter/src/main/scala/com/evolutiongaming/jsonitertool/PlayJsonJsoniter.scala

```diff
@@ -0,0 +1,107 @@
+package com.evolutiongaming.jsonitertool
+
+import com.github.plokhotnyuk.jsoniter_scala.core._
+import play.api.libs.json._
+
+import scala.collection.IndexedSeq
+
+object PlayJsonJsoniter {
+
+  implicit val jsValueCodec: JsonValueCodec[JsValue] =
+    new JsonValueCodec[JsValue] {
+
+      private val playJsonSettings = JsonParserSettings.settings.bigDecimalParseSettings
```

Comment:
rename `playJsonSettings` to `bigDecimalParseSettings`, but I would local scope it in method with super short name
like
```
val s = JsonParserSettings.settings.bigDecimalParseSettings
JsNumber(in.readBigDecimal(null, s.mathContext, s.scaleLimit, s.digitsLimit))
```
wdyt?